src/store/metadata/oxigraph/store.py

```diff
@@ -145,11 +145,38 @@ def get_topics(self) -> Optional[Dict]:  # pragma: no cover
         """
         raise NotImplementedError
 
-    def get_topic(self) -> Optional[Dict]:  # pragma: no cover
+    def get_topic(self, topic_id: str) -> Optional[Dict]:  # pragma: no cover
         """
-        Get a specific topic
+        Get a specific topic by topic_id
         """
-        raise NotImplementedError
+        # Populate the graph from the database
+        graph = self.db
+
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a single Graph of the
+        # data we need.
+        result: Graph = (
+            construct_dataset_topic_by_id(graph, topic_id)
+            + construct_dataset_subtopics_by_id(graph, topic_id)
+            + construct_dataset_parent_topics_by_id(graph, topic_id)
+        )
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "dcat:theme"}
+        )
+
+        # Use a remote context
+        data["@context"] = "https://data.ons.gov.uk/ns#"
```

Comment:
this can go. If we _were_ using context it'd need to be against `result["@context"]`, but given the schemas need updating for a context (which we'll need, but not right now) that'll keep for another piece of work, you can just delete.